web/src/components/ChatSecurityInformation.tsx

```diff
@@ -26,15 +26,15 @@ const SecurityIcon = styled(Icon)`
   color: ${props => props.theme.colors.textSecondaryColor};
 `
 
-const InformationTooltipContainer = styled.div`
+const InformationTooltipContainer = styled.div<{ $isRtl: boolean }>`
   position: absolute;
   z-index: 2000;
   border: 1px solid;
   background-color: ${props => props.theme.colors.textSecondaryColor};
   color: white;
   padding: 12px;
   text-align: center;
-  transform: translate(-95%, -90%);
+  transform: ${props => (props.$isRtl ? 'translate(95%, -90%)' : 'translate(-95%, -90%)')};
```

Comment:
🔧 There is no need to explicitly pass `isRtl` to styled components, you can just use `props.theme.contentDirection`:
```suggestion
  transform: ${props => (props.theme.contentDirection === 'rtl' ? 'translate(95%, -90%)' : 'translate(-95%, -90%)')};
```

Reply:
I remember checking contentDirection and it didn't work 😂 I thought it was removed cuz react does it automatically.